numpy/lib/_arraysetops_impl.py

```diff
@@ -248,6 +249,10 @@ def unique(ar, return_index=False, return_inverse=False,
     >>> a = np.array([[1, 1], [2, 3]])
     >>> np.unique(a)
     array([1, 2, 3])
+    >>> a = np.array([np.nan,0,0,np.nan])
+    >>> u = np.unique(a, axis=0, equal_nan=True)
+    >>> u
+    array([ 0., nan])
```

Comment:
```suggestion
```
Sorry, thanks for the effort of adding docs.  But in the end, I don't think it is helpful as is, since `equal_nan=True` is the default anyway.  It might be helpful as a brief note that it doesn't always work, but that's not it.

If you much prefer adding it, let's follow up in a second PR.